lib/rubocop/cop/rails/link_to_blank.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for calls to `link_to` that contain a
+      # `target: '_blank'` but no `rel: 'noopener'`. This can be a security
+      #  risk as the loaded page will have control over the previous page
+      # and could change its location for phishing purposes.
+      #
+      # @example
+      #   # bad
+      #   link_to 'Click here', url, target: '_blank'
+      #
+      #   # good
+      #   link_to 'Click here', url, target: '_blank', rel: 'noopener'
+      class LinkToBlank < Cop
+        MSG = 'Specify a `:rel` option containing noopener.'.freeze
+
+        def_node_matcher :blank_target?, <<-PATTERN
+          (pair {(sym :target) (str "target")} (str "_blank"))
+        PATTERN
+
+        def_node_matcher :includes_noopener?, <<-PATTERN
+          (pair {(sym :rel) (str "rel")} (str #contains_noopener?))
+        PATTERN
+
+        def on_send(node)
+          return unless node.method?(:link_to)
+
+          option_nodes = [node.children.last, node.children[3]].compact
+
+          option_nodes.map(&:children).each do |options|
```

Comment:
You can go with `#each_pair` here, instead of `map(&:children)`.

Reply:
Unfortunately at this point `option_nodes` is an `Enumerator` of Hash nodes